msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -137,6 +137,116 @@ async def on_message_activity(self, turn_context: TurnContext):
 
 ---
 
+## Receive a read receipt
+
+You can configure your bot to receive read receipt events through the **Read receipts** setting. The read receipt event helps you enhance user experience in the following ways:
```

Comment:
What is the "read receipt setting"? I think this could be made more clear.

Reply:
Updated